code/extraction.py

```diff
@@ -561,7 +560,7 @@ def contour_video(
 
 
 if __name__ == "__main__":
-    start_time = dt.now(tz.utc)
+    start_time = dt.now()
```

Comment:
As metioned on Teams, we should probably use `dt.now().astimezone()` (2024-10-16 11:45:27.378264-07:00) or `dt.now().astimezone().strftime('%Y-%m-%d %H:%M:%S %Z')` (2024-10-16 11:45:27 PDT) or `dt.now().astimezone().strftime('%Y-%m-%d %H:%M:%S.%f %Z')` (2024-10-16 11:45:27.378338 PDT)